stats/kernel-selftests

```diff
@@ -730,37 +730,37 @@ class ResctrlStater < Stater
       @test_script = $1
     when /# Pass: Check\s+(.*)/
       # Pass: Check MBM diff within 5%
-
       # The following line is repeated 3 times
       # Pass: Check resctrl mountpoint "/sys/fs/resctrl" exists
-      stats.add "#{@test}.#{$1}", 'pass' unless stats.key? "#{@test}.#{$1}"
+      add_cat_results('pass', $1, stats)
     when /# SKIP\s+(.*)/
       # The following line is repeated 3 times
       # ok 4 # SKIP Hardware does not support CAT or CAT is disabled
-      stats.add "#{@test}.#{@test_script}", 'skip' unless stats.key? "#{@test}.#{@test_script}"
-    when /^not ok\s+(.*)/
-      # not ok CAT: cache miss rate within 4%
-      @test_script = $1
-
-      # not ok 3 CMT: test
-      @test_script = $1.split(' ', 2)[1] if %w{1 2 3 4}.include? $1.split(' ', 2)[0]
+      add_cat_results('skip', @test_script, stats)
+    when /^\b(.*ok)\s+(.*)/
```

Comment:
thanks, what is \b used for here? Can it be more explicit to match like /^(not ok|ok)\s+(.*)/?

Reply:
the \b was used for the word boundary until `ok`, but yes, we can change it to be more explicit 😄